### PR TITLE
Change description and add source to metadata

### PR DESCRIPTION
**Pull request by arturo-seijas**
Metadata changes to adapt to discourse as per https://discourse.charmhub.io/t/changes-to-charmhub-io-pages/9366

### CODE REVIEW

**Review by amandahla — DISMISSED** [review]
LGTM

**Review by jdkandersson — DISMISSED** [review]

File: metadata.yaml [hunk 1]
```diff
@@ -1,11 +1,30 @@
 name: indico
 display-name: Indico
 summary: Indico web application.
+description: |
+  A Juju charm deploying and managing Indico on Kubernetes. Indico is an
```
Comment (gregory-schiano) [inline metadata.yaml:5]: Should we put links to Juju website ?

Comment (gregory-schiano) [reply metadata.yaml:5]: And indico eventually ?

File: metadata.yaml [hunk 2]
```diff
@@ -1,11 +1,30 @@
...
+  open-source tool for event organisation, archival and collaboration.
+
+  This charm simplifies initial deployment and "day N" operations of Indico
+  on Kubernetes, such as scaling the number of instances, integration with SSO,
+  access to S3 for redundant file storage and more. It allows for deployment on
+  many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
+  [Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
+  offerings.
```
[the remark this answers is not in the document]
Comment (arturo-seijas) [reply metadata.yaml:14]: Any suggestion on how to rephrase?

**Review by amandahla — DISMISSED** [review]
LGTM
(any idea why indico_rock is failing?)

File: metadata.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (merkata) [reply metadata.yaml:14]: I'd replace the second 'to' with an 'and', at least to me it sounds good.

**Review by amandahla — APPROVED** [review]
LGTM

**Review by nrobinaubertin — APPROVED** [review]

**Pull request merged**